src/spaced/spaced/game/world_spec.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <djson/json.hpp>
+#include <vector>
+
+namespace spaced {
+struct WorldSpec {
+	struct Player {
```

Comment:
Should this be something like `PlayerSpec` to avoid confusion? Although I suppose it is a bit redundant

Reply:
Yeah, it could be argued that way for sure. I didn't use the suffix because like you said it's already in the enclosing class (`WorldSpec::Player`).